### DISCUSSION TITLE
Language list should be in list each thiere language

### DISCUSSION THREAD

**Discussion by Monirzadeh** [koreader/koreader#11682] [Ideas]
Hi
i am working on translating koreader and see language name is for translation.
i think the correct way is to each language name be in that language instead of translation of that in English.
for example if you don't know English you should can find you language with your alphabet instead of equivalent of that in English. 
for example Japanese should see there language name in their kanji instead of English alphabet  in language list regardless current koreader language settings.

it is just about language selection of koreader

**Comment by Frenzie** [comment]
The languages themselves are never changed in the language selection menu for that reason. The translation is for other places where it does make sense. (Like "translate from x to y".)
